RELEASE/scripts/autoscend.ash

```diff
@@ -13977,7 +13831,16 @@ boolean doTasks()
 
 	basicAdjustML();
 	powerLevelAdjustment();
-	handleFamiliar("item");
+	if (is100FamiliarRun())
+	{
+		// re-equip a familiar if it's a 100% run just in case something unequipped it
+		// looking at you auto_maximizedConsumeStuff()...
+		handleFamiliar(to_familiar(get_property("auto_100familiar")));
```

Comment:
Is this an issue in 100% Community Service runs too?

Reply:
I don't know, I've never run CS. 
It's definitely a problem with the consumption stuff (which I tried to add a fix for too, see the change in auto_cooking.ash) as it hit me in my current Normal KoE test run which I intentionally attempted to test a 100% familiar run to make sure there were no issues.
I figure a belt and braces approach isn't going to hurt though.
I plan on cleaning up all the familiar code eventually anyway as it's a bit of a scattershot approach currently & we could do much better.